projects/ngneat/error-tailor/src/lib/control-error.directive.ts

```diff
@@ -43,7 +45,7 @@ const errorTailorClass = 'error-tailor-has-error';
     '[formControlName]:not([controlErrorsIgnore]), [formControl]:not([controlErrorsIgnore]), [formGroup]:not([controlErrorsIgnore]), [formGroupName]:not([controlErrorsIgnore]), [formArrayName]:not([controlErrorsIgnore]), [ngModel]:not([controlErrorsIgnore])',
   exportAs: 'errorTailor',
 })
-export class ControlErrorsDirective implements OnInit, OnDestroy {
+export class ControlErrorsDirective implements OnInit, OnDestroy, DoCheck {
   @Input('controlErrors') customErrors: ErrorsMap = {};
```

Comment:
What do u think about converting the inputs to signal inputs?

Reply:
Would be nice indeed! How about doing it in a separate PR, though? I believe we would also need to update the Angular version. `input` seems to be exported from v17.2.0 on, unless I'm missing something.

Reply:
Sure